### ISSUE TITLE
Mouse moving on its own in Map Editor

### ISSUE THREAD

**Issue by ronnyfugmann** [sirjuddington/SLADE#866]
**SLADE Version**: 3.1.2 beta 5
**Operating System**: Windows 10

**Issue Details**:
In 3D view, after moving the mouse downwards or to the right, panning on screen doesn't stop when the physical mouse is no longer moved (slowly continues in same direction instead).
Moving the mouse upwards or to the left works as usual though.

Tried two different mice and previous SLADE version to ensure it's not a hardware issue:
- error *did* occur with either mouse
- error *didn't* occur in previous SLADE version (tested in 3.0.1)

Mice used:
- Razer Naga Epic Chroma (cordless)
- Logitech M110 (corded)

**Comment by sirjuddington** [comment]
Hmm, never seen anything like this before, I wonder if it's some kind of DPI issue. Does it still happen in the latest [dev build](http://slade.mancubus.net/dev/build/)?

**Comment by ronnyfugmann** [comment]
I tried the latest dev build you linked to, it worked fine. (frame rate was a bit jittery though)

On a hunch (there was a major Windows update recently, one the system keeps constantly nagging you about - nothing else changed on my computer), I tried the original SLADE version where I encountered the bug before and weirdly enough it worked fine, too. (no frame rate issues here)

So for now, it seems the issue fixed itself.

I remember one of those pesky Windows updates introducing major graphics card problems a few months back (inexplicable infrequent and very annoying frame rate drops to below 30 fps in games  (when running smoothly at way above 60 fps otherwise) while any YouTube or Netflix video was open or running in Google Chrome on another monitor for example - this has not been fixed since) - maybe the mouse issue was related to that update somehow.

If I run into it again, I'll be back here. Thanks.
